tools/release/rollback.sh

```diff
@@ -63,3 +64,12 @@ gcloud docker -- push gcr.io/${PROJECT_ID}/datalab-gateway:latest
 echo "Releasing the Datalab image: ${DATALAB_IMAGE}"
 docker tag -f ${DATALAB_IMAGE} gcr.io/${PROJECT_ID}/datalab:local
 gcloud docker -- push gcr.io/${PROJECT_ID}/datalab:local
+
+echo "Pulling the rollback GPU images: ${DATALAB_GPU_IMAGE}"
+# This will fail and exit if the previous GPU image doesn't exist.
+# This will happen if we try to rollback the first GPU release, and
+# that is fine since there is nothing to rollback to.
+gcloud docker -- pull ${DATALAB_GPU_IMAGE}
```

Comment:
I worry about the following scenario:

1. We do a new release with the first GPU image.
2. We have to roll that release back
3. The rollback gets to this step and fails
4. The Jenkins job performing the rollback shows up as a failure
5. The release engineer retries the rollback, causing us to rollback one more release than intended.

How about just adding a `|| exit 0` to the end of this line?